[2022-03-16 02:52:28,834] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2022-03-16T02:52:28.129504+00:00 [queued]>
[2022-03-16 02:52:28,846] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2022-03-16T02:52:28.129504+00:00 [queued]>
[2022-03-16 02:52:28,847] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-16 02:52:28,848] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-16 02:52:28,849] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-16 02:52:28,861] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): load_seed_data_once> on 2022-03-16 02:52:28.129504+00:00
[2022-03-16 02:52:28,866] {standard_task_runner.py:52} INFO - Started process 550 to run task
[2022-03-16 02:52:28,870] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '1_init_once_seed_data', 'load_seed_data_once', 'manual__2022-03-16T02:52:28.129504+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/init.py', '--cfg-path', '/tmp/tmp15jcfxzx', '--error-file', '/tmp/tmp3yt5jqbl']
[2022-03-16 02:52:28,871] {standard_task_runner.py:77} INFO - Job 48: Subtask load_seed_data_once
[2022-03-16 02:52:28,935] {logging_mixin.py:109} INFO - Running <TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2022-03-16T02:52:28.129504+00:00 [running]> on host b1edfeccb8dc
[2022-03-16 02:52:29,038] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=1_init_once_seed_data
AIRFLOW_CTX_TASK_ID=load_seed_data_once
AIRFLOW_CTX_EXECUTION_DATE=2022-03-16T02:52:28.129504+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-16T02:52:28.129504+00:00
[2022-03-16 02:52:29,039] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-16 02:52:29,040] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /dbt && dbt seed --profiles-dir .']
[2022-03-16 02:52:29,053] {subprocess.py:85} INFO - Output:
[2022-03-16 02:52:33,493] {subprocess.py:89} INFO - 02:52:33  Running with dbt=1.0.0
[2022-03-16 02:52:33,625] {subprocess.py:89} INFO - 02:52:33  Unable to do partial parsing because of a dbt version mismatch. Saved manifest version: 1.0.3. Current version: 1.0.0.
[2022-03-16 02:52:36,596] {subprocess.py:89} INFO - 02:52:36  Found 5 models, 0 tests, 0 snapshots, 0 analyses, 581 macros, 0 operations, 3 seed files, 0 sources, 0 exposures, 0 metrics
[2022-03-16 02:52:36,600] {subprocess.py:89} INFO - 02:52:36
[2022-03-16 02:52:38,355] {subprocess.py:89} INFO - 02:52:38  Concurrency: 200 threads (target='dev')
[2022-03-16 02:52:38,357] {subprocess.py:89} INFO - 02:52:38
[2022-03-16 02:52:38,408] {subprocess.py:89} INFO - 02:52:38  1 of 3 START seed file public.bookings_1........................................ [RUN]
[2022-03-16 02:52:38,409] {subprocess.py:89} INFO - 02:52:38  2 of 3 START seed file public.bookings_2........................................ [RUN]
[2022-03-16 02:52:38,410] {subprocess.py:89} INFO - 02:52:38  3 of 3 START seed file public.customers......................................... [RUN]
[2022-03-16 02:52:40,532] {subprocess.py:89} INFO - 02:52:40  1 of 3 OK loaded seed file public.bookings_1.................................... [INSERT 7 in 2.12s]
[2022-03-16 02:52:40,545] {subprocess.py:89} INFO - 02:52:40  2 of 3 OK loaded seed file public.bookings_2.................................... [INSERT 7 in 2.13s]
[2022-03-16 02:52:40,700] {subprocess.py:89} INFO - 02:52:40  3 of 3 OK loaded seed file public.customers..................................... [INSERT 2 in 2.28s]
[2022-03-16 02:52:40,734] {subprocess.py:89} INFO - 02:52:40
[2022-03-16 02:52:40,736] {subprocess.py:89} INFO - 02:52:40  Finished running 3 seeds in 4.13s.
[2022-03-16 02:52:40,767] {subprocess.py:89} INFO - 02:52:40
[2022-03-16 02:52:40,768] {subprocess.py:89} INFO - 02:52:40  Completed successfully
[2022-03-16 02:52:40,770] {subprocess.py:89} INFO - 02:52:40
[2022-03-16 02:52:40,772] {subprocess.py:89} INFO - 02:52:40  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2022-03-16 02:52:42,197] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-16 02:52:42,234] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=1_init_once_seed_data, task_id=load_seed_data_once, execution_date=20220316T025228, start_date=20220316T025228, end_date=20220316T025242
[2022-03-16 02:52:42,268] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-16 02:52:42,295] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
